### ISSUE TITLE
Pathway to finishing the website

### ISSUE THREAD

**Issue by postpolar** [apesonline-gg/AIPs#3]
- [ ] I propose we formalise a path toward building the website. 

We focus on:
(1) The content of the site: white paper and art sections.
(2) Web3 devs select **one** dApp they know they can get working by launch, e.g. the MODEM forum or an NFT. 
(3) We create the concept art for future dApps, e.g. Winamp, the browser, FMK.

**Comment by v-stickykeys** [comment]
Yeah totally agree with this plan. In terms of 2 I think the 3D NFTs are a good one to engage the community, also after chatting with secretwaves it seems likely we can get this done. We can tie that to chat if there's someone who can handle that side.

**Comment by postpolar** [comment]
Yep, I think we should make the 3D NFT the priority dApp.
